docs/webhooks.md

```diff
@@ -0,0 +1,72 @@
+# Webhooks
+
+?> Available `5.2.0`
+
+A variety of webhooks are available to extend the functionality of SQLPad.
+
+To enable a specific webhook, provide a URL for the event you would like to subscribe to. This URL will be sent an HTTP(S) POST with a payload of relevant information when the event occurs.
+
+To ensure a webhook call is valid, `SQLPAD_WEBHOOK_SECRET` may be configured, and validated on every webhook call.
+
+Each webhook will contain the following headers:
+
+- `Content-Type`: `application/json`
+- `SQLPad-Secret`: `SQLPAD_WEBHOOK_SECRET value`
+- `SQLPad-URL`: `http://your-sqlpad-public-url:port/base-url`
```

Comment:
Is this actually the content of PUBLIC_URL env var?

Reply:
yeah it'd be `PUBLIC_URL` + `:SQLPAD_PORT` (if not standard) + `SQLPAD_BASE_URL`